sphinx/README.md

```diff
@@ -6,23 +6,29 @@ You can build the tutorial yourself if you wish.
 
 ## Install Prerequisites
 
+Generate the [doxygen docs](https://agisostack-plus-plus.readthedocs.io/en/latest/Developer%20Guide.html#doxygen):
+
+```bash
+doxygen doxyfile
+```
+
+Install required python modules:
 ```bash
-pip install sphinx
-pip install sphinx-rtd-theme
+pip install -r requirements.txt
 ```
 
 ## Build the Tutorial
 
 Windows:
 
 ```bash
-./make html
+./make.bat html
 ```
 
 Linux:
 
 ```bash
-make html
+sphinx-build -M html source build
 ```
 
-Then, you can view the documentation at build\html\index.html in your web browser.
+Then, you can view the documentation at `sphinx_root\build\html\index.html` in your web browser.
```

Comment:
```suggestion
Then, you can view the documentation at `sphinx\build\html\index.html` in your web browser.
```
Good addition! Small suggestion: since `sphinx` is our `sphinx_root`, I guess this makes sense to just put that in here?